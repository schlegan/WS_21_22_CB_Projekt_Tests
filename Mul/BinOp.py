print(69 * (1+2)) # 207
print(69 * (5-2)) # 207
print(40 + (10 * 5)) # 90
print(40 - (20 * 5)) # -40
