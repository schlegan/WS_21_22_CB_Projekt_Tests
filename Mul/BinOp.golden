20720790-40
